ide/nativeimage.api/src/org/netbeans/modules/nativeimage/api/debug/NIDebugger.java

```diff
@@ -257,6 +257,16 @@ public String getVersion() {
         return provider.listVariables(name, includeNondebug, maxResults);
     }
 
+    /**
+     * Get a list of source paths in the debuggee.
+     *
+     * @param nameRegexp regular expression that filters the list of source paths
+     * @since 0.7
+     */
+    @CheckForNull
+    public List<SourceInfo> listSources(String nameRegexp) {
```

Comment:
A note in `apichanges.xml` would be good. This method shall never return `null`, rather empty list - unless you want to use `null` for some use-case. In such case mention that in Javadoc.

Reply:
The API changes were abandoned for now.